lib/lltz_ir/dsl.ml

```diff
@@ -86,7 +86,7 @@ module T = struct
   let chain_id_prim = LE.{ desc = Prim (LP.Chain_id, []); range = dummy  }
```

Comment:
About the dummies: try and add a parameter `range`. (I am not a fan of an optional parameter when it requires an extraneous `()` parameter to be able to be erased.)

Reply:
rebased and added range params